apiml-security-common/src/main/java/org/zowe/apiml/security/common/error/AuthExceptionHandler.java

```diff
@@ -151,4 +153,13 @@ private void handleAuthenticationException(HttpServletRequest request, HttpServl
         final HttpStatus status = HttpStatus.INTERNAL_SERVER_ERROR;
         writeErrorResponse(message, status, response);
     }
+
+    private void handleServiceNotAccessibleException(HttpServletRequest request, HttpServletResponse response, RuntimeException ex) throws ServletException {
+        log.debug(ERROR_MESSAGE_500, ex.getMessage());
+        log.debug("", ex);
```

Comment:
I was just replicating how it was done in the previous method, but I guess it doesn't make much sense, I will just add the exception instead of just the message.